index.html

```diff
@@ -3,13 +3,25 @@
 		<title> Westie World </title>
 		<link rel="stylesheet" type="text/css" href="stylesheet.css" />
 		<link rel="stylesheet" type="text/css" href="//fonts.googleapis.com/css?family=Open+Sans" />
+		<!-- <link rel="stylesheet" type="image" href="public/westieWorld.png"/> -->
+		<!-- <link rel="stylesheet" type=> -->
 	</head>
 	<body>
-		<div class="header">
-			<h1>Westie World!</h1>
+		<div class="header" height="400px" width="400px">
+			<!-- <h1>Westie World!</h1> -->
+			<img class="headerImg" src="public/westieWorld.png" alt="Westie World Header" style="width:400px;height:400px;">
 		</div>
 		<div class="about">
-			This event is super awesome! We have lots of amazing workshops and awesome staff! Whoo!
+			<p class="about-p1">Welcome to Westie World, a world first West Coast Swing theme park! Our theme park is populated by one-of-a-kind androids that are built to match both WCS competitors and instructors more accurately than ever before. You'll feel like you're really at a dance competition, the realism can't be matched by anyone else! At our theme park, you can live out your dance dreams such as:</p>
```

Comment:
Is there something more descriptive we can use instead of `about-p1` and `about-p2`